src/pages/index.tsx

```diff
@@ -79,11 +79,11 @@ const Home: NextPage<Props> = ({ events: events }: Props) => {
             </a>
           </div>
         )}
+        <div className='mb-2' />
+        <Sponsors />
         <Newsletter />
+        <Community members={communityMembers} />
```

Comment:
we might want to limit showed members in the home page, so to not increase main page height too much

Reply:
Yes, it makes sense, as reported in the PR description

# 📝 To Do
- when the list of members grows, maybe we just need to show them in chunks of X elements

Could be a good idea to leave it as an issue and implement it when we really need it? I didn't to in this PR to limit its size and to avoid premature optimization

Reply:
oh sorry, didn't pay the right attention to it